spark/src/main/scala/geotrellis/spark/ingest/Ingest.scala

```diff
@@ -66,7 +66,7 @@ object Ingest {
     (sink: (RasterRDD[K], Int) => Unit): Unit =
   {
     val (_, rasterMetaData) =
-      RasterMetaData.fromRdd(sourceTiles, destCRS, layoutScheme)(_.projectedExtent.extent)
+      RasterMetaData.fromRdd(sourceTiles, destCRS, layoutScheme)
```

Comment:
think we can not to merge #1338 as it breaks compilation anyway until further investigation, but there we have to use: `RasterMetaData.fromRdd(sourceTiles, layoutScheme)` and this breaks `LatLng` `Ingest` tests .